keyboards/cool836A/info.json

```diff
@@ -0,0 +1,19 @@
+{
+    "keyboard_name": "cool836A",
+    "url": "https://github.com/telzo2000/cool836A",
+    "maintainer": "Ohashi",
+    "width": 12,
+    "height": 3,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
```

Comment:
You'll probably need to change this, too.

Reply:
I'm afraid I don't know what to edit.

If you mean this area should be changed,
```
    "layouts": {
        "LAYOUT": {
            "layout": [
```
I'd like to say I just followed the default `keymap.c`, which is automatically generated when you run
```
$ bash util/new_keyboard.sh
```

This `info.json` properly tells cool836A is a 3x12 keyboard, and clearly shows each position of keyswitches.

Reply:
Is it okay that I do nothing about this?

Reply:
You'd want to re-order the keys in this, since it uses the LAYOUT macro in the keyboard's h file to determine the order of these keys.  Since those have changed... this would need to change. 

So, I'd say no to leaving it. But ripping it out may be okay for now. 

However, you could use KLE->kbfirmware and then [this](https://noroadsleft.github.io/kbf_qmk_converter/) to generate a new, accurate info.json.

Reply:
nevermind. Looks like you've updated the JSON, and it works properly in fast input on the configurator.